js/main.js

```diff
@@ -1,12 +1,63 @@
 /* eslint-disable max-classes-per-file */
 document.addEventListener('DOMContentLoaded', () => {
-  const addBtn = document.querySelector('.add');
-  const bookTitle = document.getElementById('title');
-  const bookAuthor = document.getElementById('author');
-  const bookListWrap = document.querySelector('.book-ul');
   const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
   let removeBtn = [];
+  const mainWrap = document.querySelector('main.our-content');
+  const navItems = [...document.querySelectorAll('.nav-list li a')];
+
+  class Screens {
+    screens = [
+      `<div class="book-list main-content display-page">
+                  <h1>All Awesome Books</h1>
+                  <ul class="book-ul">
+                  </ul>
+                </div>`,
+      `<div class="add-new main-content">
+                  <form>
+                    <input type="text" id="title" placeholder="title" required minlength="3">
+                    <input type="text" id="author" placeholder="author" required minlength="3">
+                    <button class="add">Add</button>
+                  </form>
+                </div>`,
+      `<div class="contact-info main-content">
+                  <h1>Contact Information</h1>
+                  <div class="contact">
+                    <p>Do you have any questions or just want to say hello?
+                      <span>You can reach out to us!</span>
+                    </p>
+                    <ul>
+                      <li>our email mail@email.com</li>
+                      <li>our phone number: 004354345345234</li>
+                      <li>our address: Streetname 22, 84604 City,Country
+                      </li>
+                    </ul>
+                  </div>
+                </div>`,
+    ]
+  }
+
+  const screens = new Screens();
+  screens.screens.forEach((screen) => {
+    mainWrap.innerHTML += screen;
+  });
 
+  const screenArr = [...document.querySelectorAll('.main-content')];
+
+  function display(content) {
+    navItems.forEach((item) => item.parentElement.classList.remove('active'));
+    navItems[content].parentElement.classList.add('active');
+    screenArr.forEach((item) => {
+      item.classList.remove('display-page');
+      screenArr[content].classList.add('display-page');
+    });
+  }
+  navItems.forEach((item, i) => {
+    item.addEventListener('click', () => display(i));
+  });
+  const bookListWrap = document.querySelector('.book-ul');
+  const addBtn = document.querySelector('.add');
+  const bookTitle = document.getElementById('title');
+  const bookAuthor = document.getElementById('author');
```

Comment:
- [ ] In the `function implementTime`, as required kindly use the [Luxon](https://moment.github.io/luxon/docs/manual/install.html) library. See [Get the current date and time](https://moment.github.io/luxon/docs/manual/tour.html#get-the-current-date-and-time).

Reply:
* I think while trying to fix the previous TSE requested changes you broke your code as the navigation links don't work please fix it 👍